manifests/base/argo-rollouts-deployment.yaml

```diff
@@ -15,6 +15,8 @@ spec:
       labels:
         app.kubernetes.io/name: argo-rollouts
     spec:
+      nodeSelector:
+        beta.kubernetes.io/os: linux
```

Comment:
We can't not enforce these labels here:
1. this specific label has already been deprecated in favor of `kubernetes.io/os`
2. this label is too opinionated may not be consistent across kubernetes installations

Reply:
Its a tough one for sure and I use kubernetes.io/os in my own version, without this though you'll find that this is broken for any mixed os cluster.

If this isn't added then much like the getting started in the readme there should be a note for mixed OS clusters manually adding the nodeSelector on their own?

Reply:
[kustomize](https://kustomize.io/) is probably what you are looking for to apply your additional requirements to the base install.